app/dist/index.html

```diff
@@ -7,18 +7,19 @@
   <link rel="stylesheet" href="./lib/common.css">
   <script type="text/javascript">
     window.$ = window.jQuery = require('jquery');
+
     // DevTools
     document.addEventListener('keydown', function (e) {
-      if (e.which === 123 /* F12 */) {
+      // F12
+      if (e.which === 123) {
         require('electron').remote.getCurrentWindow().toggleDevTools();
-      } else if (e.which === 116 /* F5 */) {
```

Comment:
could you explain any reason to remove this? 😄

Reply:
in my local version, when I pressed F5, the window would reload but it would be stuck loading results indefinitely, not sure if you've seen the same behaviour?

Reply:
Reverted as it's out of scope for theming.